app/code/Magento/Sales/Block/Order/Email/Items/Order/DefaultOrder.php

```diff
@@ -93,10 +95,10 @@ public function getProductAdditionalInformationBlock()
     /**
      * Get the html for item price
      *
-     * @param OrderItem $item
+     * @param OrderItem|InvoiceItem|CreditmemoItem  $item
      * @return string
      */
-    public function getItemPrice(OrderItem $item)
+    public function getItemPrice($item)
```

Comment:
Unfortunately, we can't change methods signature due to [Backward Compatible Development Policy](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/). 
Please, add the new method, refactor all usages and deprecate the old method.

Reply:
It's a normalization, as https://github.com/AurelienLavorel/magento2/blob/3311bf9bf619e8dc04644180ae800bd31cc30018/app/code/Magento/Sales/Block/Order/Email/Items/DefaultItems.php#L101
I'm not breaking a fonctionnality...

Reply:
@sidolov ?

Reply:
@sidolov I made some changes to not break override, fine for you?